Formula/r/rbenv.rb

```diff
@@ -1,11 +1,16 @@
 class Rbenv < Formula
   desc "Ruby version manager"
-  homepage "https://github.com/rbenv/rbenv"
-  url "https://github.com/rbenv/rbenv/archive/refs/tags/v1.2.0.tar.gz"
-  sha256 "3f3a31b8a73c174e3e877ccc1ea453d966b4d810a2aadcd4d8c460bc9ec85e0c"
+  homepage "https://rbenv.org"
+  url "https://github.com/rbenv/rbenv/archive/refs/tags/v1.3.0.tar.gz"
+  sha256 "7e49e529ce0c876748fa75a61efdd62efa2634906075431a1818b565825eb758"
   license "MIT"
   head "https://github.com/rbenv/rbenv.git", branch: "master"
 
+  livecheck do
+    url :stable
+    strategy :github_latest
+  end
+
```

Comment:
Thanks for the PR @mislav! Are you planning on using GitHub pre-releases? If not, we can omit this and livecheck will use the git tags automatically.